src/SDKs/DataFactory/Management.DataFactory/Generated/SdkInfo_DataFactoryManagementClient.cs

```diff
@@ -39,10 +39,10 @@ internal static partial class SdkInfo
       // BEGIN: Code Generation Metadata Section
```

Comment:
If there was no new code generated, please revert this and the .txt file.

Reply:
reverted, thanks!